nautobot/dcim/models/locations.py

```diff
@@ -114,11 +114,29 @@ def clean(self):
 
 
 class LocationQuerySet(TreeQuerySet):
+    @classmethod
+    def as_manager(cls, with_tree_fields=False):
+        """Get an appropriate TreeManager subclass for working with Locations."""
+        LocationManager = TreeManager.from_queryset(cls)
+        LocationManager._built_with_as_manager = True
+        LocationManager._with_tree_fields = with_tree_fields
+        return LocationManager()
```

Comment:
Need to revisit this - I don't think this override of `as_manager` *should* be needed but I was running into issues without it early on.